repository/src/main/java/org/apache/atlas/repository/store/graph/v1/DeleteHandlerV1.java

```diff
@@ -1278,6 +1278,15 @@ private void setBlockedClassificationIds(AtlasEdge edge, List<String> classifica
         }
     }
 
+    public void createClassificationOnlyPropagationDeleteTasksAndQueue(List<AtlasEdge> classificationEdges, Set<String> deletedEdgeIds) {
+        for (AtlasEdge classificationEdge : classificationEdges) {
+            String classificationVertexId = classificationEdge.getInVertex().getIdForDisplay();
```

Comment:
Use GraphHelper.getPropagatableClassifications function to get the prpagable classifications

Reply:
GraphHelper.getPropagatableClassifications() contains search functionality inside. Since we already have the edges and connected vertices that we are looking for, I do not think we should make use of this function. It will add additional cost.

Reply:
The problem here is another, you are only fetching the classification from in vertices but the tag propagation in edges could be different. In case of `TWO_TO_ONE` tag propagation classification propagated from different direction inVertex -> OutVertex so that case is handled there also classification could be propagated to both direction even None. All the cases are handled there. Again you have to handle those cases. e.g
there is an edge A->B with tagPropagation as ONE_TO_TWO then you check the classification from in vertex A no problem, but if it is TWO_TO_ONE then you need to collect the  classification from out vertex B if tag Propagation is BOTH then we need to collect from both A,B and also if tag propagation is none then we don't collect anything. FYI Process Input -> Asset is TWO_TO_ONE tag propagation, that would be affected in that case.

Reply:
So should I manually handle?